src/viam/sdk/common/linear_algebra.hpp

```diff
@@ -0,0 +1,10 @@
+#pragma once
+#include <boost/numeric/ublas/storage.hpp>
+namespace viam {
+namespace sdk {
+
+/* typedef boost::numeric::ublas::bounded_array<double, 3> Vector3; */
+typedef std::array<double, 3> Vector3;
```

Comment:
It's not immediately clear to me what the advantage of using the `boost` type is. Can you go into some more detail about what you think the `boost` type gives us that the `std::array` doesn't?

Reply:
In my mind, the benefit of the `boost` type is that it has additional member methods that will help both us and users. If we end up needing to do vector operations, it will be much nicer to have a real vector rather than a wrapped array. 

(This also applies to users. It is very likely that they will want to do things like add / cross-multiply vectors and it is more convenient if they do not have to convert into and out of a `std::array`)

Reply:
Makes sense to me, given the above I'm happy to switch to the `boost` type.

Reply:
After more investigation, I have decided to use the `boost::qvm::vec` class. It has turned out quite well.